opal/corelib/string/encoding.rb

```diff
@@ -181,6 +181,23 @@ def each_byte(&block)
     self
   end
 
+  def each_codepoint(&block)
+    return enum_for :each_codepoint unless block_given?
+    %x{
+      for (var i = 0, length = self.length; i < length; i++) {
+        var code = self.codePointAt(i);
```

Comment:
The local variable assignment isn't necessary here.

Reply:
you mean we should `yield` the result of `codePointAt` ?

```
#{yield `self.codePointAt(i)`}
```

Reply:
Exactly.